package.json

```diff
@@ -47,11 +47,13 @@
     "jest-cli": "^27.4.5",
     "prettier": "~2.7.0",
     "puppeteer": "^19.4.1",
+    "rollup-plugin-dotenv": "^0.5.0",
```

Comment:
Is this devDependency necessary in this repo?

Reply:
yes, it is use to pick up the environment information